### DISCUSSION TITLE
Welcome to react-activation Discussions!

### DISCUSSION THREAD

**Discussion by CJY0208** [CJY0208/react-activation#83] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ysh199882** [comment]
请问有联系方式吗

**Reply by xiaohuoni** [reply]
v我 50 ，给你他所有的私人信息（0 .. 0）

**Reply by CJY0208** [reply]
？居然还有个 discussions

谨防网络诈骗 @ysh199882 微信 375564567
